spiceypy/tests/gettestkernels.py

```diff
@@ -73,12 +73,14 @@ class ExtraKernels(object):
     earthHighPerPck_url = "https://raw.githubusercontent.com/AndrewAnnex/SpiceyPyTestKernels/master/earth_031228_231229_predict.bpc"
     phobosDsk_url       = "https://raw.githubusercontent.com/AndrewAnnex/SpiceyPyTestKernels/master/phobos_lores.bds"
     marsSpk_url         = "https://raw.githubusercontent.com/AndrewAnnex/SpiceyPyTestKernels/master/mar022-1.bsp"
+    mroFk_url           = "https://naif.jpl.nasa.gov/pub/naif/MRO/kernels/fk/mro_v15.tf"
```

Comment:
ah okay I see now, yes please issue a pr against the test kernels repository adding this file to the collection